Sources/CoreLocationMiddleware/CoreLocationMiddleware.swift

```diff
@@ -43,25 +62,28 @@ public struct DeviceCapabilities: Equatable {
     public let isLocationServiceAvailable: Bool
 }
 
+// MARK: - REDUCERS
 
-let locationReducer = Reducer<LocationAction, LocationState> { action, state in
-    var state = state
-    switch action {
-    case .startMonitoring,
-         .stopMonitoring,
-         .requestAuthorizationStatus,
-         .requestAuthorizationType,
-         .requestPosition,
-         .requestDeviceCapabilities:
-        break
-    case let .gotAuthzStatus(status): state.authzStatus = status
-    case let .gotPosition(position): state.location = position
-    case .gotDeviceCapabilities, .receiveError : break
-    }
+extension Reducer where ActionType == LocationAction, StateType == LocationState {
+    static let location = Reducer<StatusAction, LocationState>.status.lift(action: \LocationAction.status)
+}
 
-    return state
+extension Reducer where ActionType == StatusAction, StateType == LocationState {
+    static let status = Reducer { action, state in
+        var state = state
+        switch action {
+        case let .gotAuthzStatus(status):
+            state.authzStatus = status.status
+            state.authzAccuracy = status.accuracy
+        case let .gotPosition(position): state.location = position
+        case .gotDeviceCapabilities, .receiveError : break
+        }
+        return state
+    }
```

Comment:
Breaking down into the "status" part allows us to only worry about those actions that impact the state. However, as per my previous comment, I am now unsure where this is actually being called...